speech_recognition/__init__.py

```diff
@@ -1390,7 +1390,38 @@ def recognize_tensorflow(self, audio_data, tensor_graph='tensorflow-data/conv_ac
             for node_id in top_k:
                 human_string = self.tflabels[node_id]
                 return human_string
+            
+    def recognize_vosk(self, audio_data, language='en'):
+        from vosk import Model, KaldiRecognizer
 
+        if not os.path.exists("model"):
+            return "Please download the model from https://github.com/alphacep/vosk-api/blob/master/doc/models.md and unpack as 'model' in the current folder."
+            exit (1)
+
+        import pyaudio
+
+        model = Model("model")
```

Comment:
Hey, you shouldn't create model for every request. Model object must be shared across requests.

Reply:
How?

Reply:
You can always edit as maintainer

Reply:
Tried it in newest commit? Is it all right now?